lxc/storage.go

```diff
@@ -88,6 +88,10 @@ lxc storage delete [<remote>:]<pool>
 lxc storage edit [<remote>:]<pool>
     Edit storage pool, either by launching external editor or reading STDIN.
 
+lxc storage <pool-name> btrfs.mount_options "<mount option>,<mount option>,<mount option>"
+    add btrfs mount options to btrfs storage pool
+    EXAMPLE: lxc storage <pool-name> btrfs.mount_options "rw,nospace_cache,lazytime,user_subvol_rm_allowed,autodefrag"
+
```

Comment:
Thanks! Just a few things here:
 - Examples must go at the end of the help message (see the existing ones below), otherwise that breaks the man pages.
 - This should be reformatted to match the existing examples.
 - We're missing the result of "make i18n && make update-po"